### DISCUSSION TITLE
I have a problem with npm run build - Add to cart, Change list

### DISCUSSION THREAD

**Discussion by rneo** [Oksydan/falcon#339] [General]
I have a problem with npm run build, something is broken in theme.js because adding to cart (popup), deleting from the cart pull-out menu, and changing the list, and possibly much more, do not work. I used the latest plugins and template. Node: v18.18.2. I've tried everything, nothing works.

**Comment by Oksydan** [comment]
What is your whole project structure? Is your theme inside project_root/themes/? Does your project root contains Falcon modules inside project_root/modules directory?

**Comment by rneo** [comment]
Tak dokładnie tak
Yes, exactly

**Comment by Oksydan** [comment]
What is your env file configuration and your environment configuration? Shop_url, physical_url?

**Reply by rneo** [reply]
OK udało się, wgrałem od zera szablon, a potem wgrałem swoje modyfikacje i teraz wszystko działa, musiał gdzieś pojawić się błąd, nie zamknięte coś, itp.

OK, it worked, I uploaded the template from scratch, and then I uploaded my modifications, and now everything works, there must have been an error somewhere, something not closed, etc.

**Comment by Oksydan** [comment]
Since it's resolved, I am closing this discussion. Feel free to open new one.
